tests/test_wallet_wallet_recovery.py

```diff
@@ -217,16 +437,38 @@ def test_resumed_flow(
                     new_owner=item["new_wallet"]["address"],
                 )
 
-        # Recovery step 2 - fail
+        # Check recovery status
+        status_response = operate_client.get(
+            url="/api/wallet/recovery/status",
+        )
+        assert status_response.status_code == 200
+        status_response = status_response.json()
+        assert status_response["prepared"] is True
+        assert status_response["bundle_id"] is not None
+        assert status_response["has_swaps"] is True
+        assert status_response["has_pending_swaps"] is True
+
+        # Check recovery funding requirements
```

Comment:
could you also check here, that make another `prepare` call returns the same bundle?